src/MatrixClientPeg.ts

```diff
@@ -464,8 +464,15 @@ class MatrixClientPegClass implements IMatrixClientPeg {
             },
         };
 
-        if (SecurityCustomisations.getDehydrationKey) {
-            opts.cryptoCallbacks!.getDehydrationKey = SecurityCustomisations.getDehydrationKey;
+        // if (SecurityCustomisations.getDehydrationKey) {
+        //     opts.cryptoCallbacks!.getDehydrationKey = SecurityCustomisations.getDehydrationKey;
+        // }
+
+        console.log("CryptoSetupExtensions: Executing getDehydrationKeyCallback...");
+        const dehydrationKeyCallback = ModuleRunner.instance.extensions.cryptoSetup?.getDehydrationKeyCallback();
+        console.log("CryptoSetupExtensions: Executing getDehydrationKeyCallback...Done");
```

Comment:
this logging looks like it will be very noisy in the common case. Perhaps we could just log something if there is a `cryptoSetup` extension, or if `getDehydrationKeyCallback` returns non-null?

Reply:
This was mostly useful while debugging, so I'll remove them. If we do need logging here, it probably makes most sense to do this from the module/extension.